components/cstudio-forms/controls/rte.js

```diff
@@ -465,7 +465,7 @@ YAHOO.extend(CStudioForms.Controls.RTE, CStudioForms.CStudioFormField, {
 	        editor_selector : rteUniqueInitClass,
 	        theme : "advanced",
 	        skin : "cstudio-rte",
-	        width : width,
+			width: '100%', // Why? Field width should be flexible, because of responsiveness, should resize based on the screen size, this works together with max width set on post render event below on setup: method 
```

Comment:
We cannot do this... width is handled by the form def.... ideally it matches the size of the region so that the wrapping of text is correct